### DISCUSSION TITLE
Portuguese translation

### DISCUSSION THREAD

**Discussion by alvsz** [aclap-dev/video-downloadhelper#1379] [Language & Translation]
[pt_BR.txt](https://github.com/user-attachments/files/17451651/pt_BR.txt)

**Comment by alalaimsocool** [comment]
[pt_BR (v9.1.1.0).txt](https://github.com/user-attachments/files/17551012/pt_BR.v9.1.1.0.txt)

**Comment by matschcar** [comment]
[pt_BR.txt](https://github.com/user-attachments/files/17592996/pt_BR.txt)

**Comment by psantanads** [comment]
[pt_BR.txt](https://github.com/user-attachments/files/17791141/pt_BR.txt)

**Comment by xtixijx** [comment]
[pt_BR.txt](https://github.com/user-attachments/files/17953566/pt_BR.txt)

Version 9.1.0.44
